corehq/apps/user_importer/helpers.py

```diff
@@ -159,6 +164,11 @@ def update_user_data(self, data, uncategorized_data, profile, domain_info):
             if value is None or value == '':
                 self.user.pop_metadata(key)
 
+        if self.user.user_data.get(PROFILE_SLUG) and self.user.user_data[PROFILE_SLUG] != current_profile_id:
+            self.logger.add_info(_("CommCare Profile: {profile_name}").format(
```

Comment:
what's the guideline for when to use `add_info` vs `add_change_message`?

Reply:
I should document this or better to standardize this with better methods in class or documentation there, my approach has been
1. `add_changes`: changes to actual user properties, for audit, ignored for new user. Example: `first_name`
2. `add_change_message`: change messages to users that are not exactly properties, ignored for new user. Example: "Password reset" / "Added phone number 9999999999"
3. `add_info`: any useful information, mainly for display, irrespective of new/old user. Example: "Role: Read Only[role-id]"